README.md

```diff
@@ -8,7 +8,7 @@ This repository is intended to be used in conjunction with other parts of the te
 library (standard, os, grid...) to actually configure a system. To help putting together
 everything needed in a way directly usable in a real configuration, it is recommended to
 follow the procedure describe at 
-https://trac.lal.in2p3.fr/Quattor/wiki/Download/QWGTemplates/Install#Upgradinganexistingrepository.
+http://www.quattor.org/documentation/2014/06/06/how-to-use-template-library.html#how-to-get-it?.
```

Comment:
I'll update the text as suggested, clearly better! the `? ` is unfortunately part of the URL... This is coming from the anchor automatically generated by Jekyl, because the title ends with a `? `. This mistake was overlooked initially, If you think this is better, I can update the referred page to remove the `?` in the title.